### ISSUE TITLE
Restrict tapping empty calendar area to some defined hours (with visual background)

### ISSUE THREAD

**Issue by piotrwojtowicz** [alamkanak/Android-Week-View#98]
Hello guys, 

do you have any tips how to solve such scenario: 

Let's say I have a hairdresser saloon opened 8-16 every day from Mon to Fri and Sat 9-16. I would like to have calendar showing week view with areas (covering saloon workhours) where customer can tap to add new visit (if no visit exists there). Tapping on such area would execute method for EmptyViewClickListener. See attached screen. 

So to sum up: how to nicely present user when he can add new event within some specified hours if no event exists there at the moment. Something like: http://i.imgur.com/qSTXlNZ.png

Any help would be appreciated :)

regards, 
Piotr

**Comment by caske33** [comment]
An idea would be to have events for empty slots (in a different color than booked slots), and when you click on them, you can say which hour you want to book.

**Comment by piotrwojtowicz** [comment]
Thank you caske33, I will give it a try :) It will be a bit of code I guess but can be done.
